### PR TITLE
[IMP] l10n_uk: Created new account tags and assing into taxes

### PR DESCRIPTION
**Pull request by sga-odoo**
Task: https://www.odoo.com/web?debug=#id=22697&view_type=form&model=project.task&action=327&menu_id=4720
PAD: https://pad.odoo.com/p/r.f47955309f3674964eaf519b6ceaf45e

### CODE REVIEW

File: addons/l10n_uk/data/account.tax.template.csv [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-PT11,PT11,l10n_uk,purchase,Standard rate purchases (20%),percent,20,2201,2201,,tag_pt11,tax_group_20
+ST0,ST0,l10n_uk,sale,Zero rated sales,percent,0,2200,2200,,"tag_uk_02,tag_uk_06",tax_group_0
+ST1,ST1,l10n_uk,sale,Standard rate sales (17.5%),percent,17.5,2200,2200,,"tag_uk_02,tag_uk_06",tax_group_175
+ST2,ST2,l10n_uk,sale,Exempt sales,percent,0,2200,2200,,"tag_uk_02,tag_uk_06",tax_group_0
+PT0,PT0,l10n_uk,purchase,Zero rated purchases,percent,0,2201,2201,,"tag_uk_04,tag_uk_08",tax_group_0
+PT1,PT1,l10n_uk,purchase,Standard rate purchases (17.5%),percent,17.5,2201,2201,,"tag_uk_04,tag_uk_08",tax_group_175
+PT2,PT2,l10n_uk,purchase,Exempt purchases,percent,0,2201,2201,,"tag_uk_04,tag_uk_08",tax_group_0
+PT8M,PT8M,l10n_uk,none,Standard rated purchases from EC,percent,17.5,2201,2201,,tag_uk_03,tax_group_0
+PT8R,PT8R,l10n_uk,none,Reverse EC Purchase VAT,percent,-17.5,2201,2201,,tag_uk_05,tax_group_0
+PT8,PT8,l10n_uk,purchase,Standard rated purchases from EC,group,,2201,2201,"PT8M,PT8R","tag_uk_09,tag_uk_03",tax_group_0
```
Comment (oco-odoo) [inline addons/l10n_uk/data/account.tax.template.csv]: Shouldn't only be tag_uk_09 ?

File: addons/l10n_uk/data/account.account.tag.csv [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
-tag_pt8m,PT8M,taxes
-tag_pt8r,PT8R,taxes
-tag_pt8,PT8,taxes
-tag_pt5,PT5,taxes
-tag_st4,ST4,taxes
-tag_pt7,PT7,taxes
-tag_st11,ST11,taxes
-tag_pt11,PT11,taxes
+tag_uk_01,Net VAT to be paid/reclaimed,taxes
+tag_uk_02,VAT on Sales and other outputs,taxes
```
Comment (oco-odoo) [inline addons/l10n_uk/data/account.account.tag.csv]: You should give better names to all your tags, as it was done in l10n_ch (here: https://github.com/odoo/odoo/blob/11.0/addons/l10n_ch/data/account_vat2011_data.xml ). Their ids should match the ones of the record lines they correspond to. 

For example, for tag_uk_02, you should name it vat_tag_0101, as it relates to line financial_report_line_uk_0101. Please do the same for all other tags.

**Comment by jco-odoo** [comment]
@oco-odoo As the task was cancelled, can you confirm this to be cancelled too?

**Comment by oco-odoo** [comment]
@jco-odoo I suppose it must be canceled too; it depenrds on the content on the new task ;)

**Comment by jco-odoo** [comment]
@oco-odoo The new task was not adapted to this.  And it would simplify the config.  So, we will need to adapt it.

**Comment by rdeodoo** [comment]
Done at https://www.odoo.com/web#id=33297&view_type=form&model=project.task&menu_id= if I am not mistaken

**Pull request closed without merging**